packages/server/tests/acceptance/rpc.spec.ts

```diff
@@ -354,6 +354,26 @@ describe('RPC Server Acceptance Tests', function () {
                     expect(balanceChange.toString()).to.eq(ONE_TINYBAR.toString());
                 });
 
+                it('should execute "eth_sendRawTransaction" and deploy a large contract', async function() {
+                    const transaction = {
+                        type: 2,
+                        chainId: Number(CHAIN_ID),
+                        nonce: await relay.getAccountNonce(accounts[2].address),
+                        maxPriorityFeePerGas: Assertions.defaultGasPrice,
+                        maxFeePerGas: Assertions.defaultGasPrice,
+                        gasLimit: defaultGasLimit,
+                        data: '0x' + '00'.repeat(5121),
+                    };
+
+                    const signedTx = await accounts[2].wallet.signTransaction(transaction);
+                    const transactionHash = await relay.call('eth_sendRawTransaction', [signedTx]);
+                    const info = await mirrorNode.get(`/contracts/results/${transactionHash}`);
+                    expect(info).to.have.property('created_contract_ids');
+                    expect(info).to.have.property('contract_id');
+                    expect(info.created_contract_ids.length).to.be.greaterThan(0);
```

Comment:
Presuming it will only contain the actual created contract then you can make this equal 1